problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -1,10 +1,81 @@
+			\item   $\mat{7\\6\\-3} \cdot \left(\mat{5\\11\\-1} + \mat{-2\\-6\\-1}\right)$
+			\item   $\mat{1\\3\\0\\-5\\5} \cdot \mat{1\\2\\2\\1\\2}$
+			\item   $\frac{1}{2}\mat{2\\5\\4} \cdot \mat{1\\0\\-1}$
 		\end{enumerate}
+		\prob Compute the length of the following vectors.
+		\begin{enumerate}
+			\item $\mat{2\\0}$
+			\item $\mat{1\\2\\3}$
+			\item $4\mat{5\\-6\\15\\2}$
+			\item $\mat{-\frac{\sqrt{10}}{10}\\-\frac{9\sqrt{10}}{50}\\\frac{12\sqrt{10}}{50}}$
+			\item $\mat{\frac{1}{3}\\\frac{\sqrt{3}}{3}\\0\\-\frac{1}{3}\\\frac{2}{3}}$
+		\end{enumerate}
+		\prob Find the angle between the following vectors.
+		\begin{enumerate}
+			\item $\mat{1\\0}$ and $\mat{-3\\4}$
+			\item $\mat{1\\0\\1}$ and $\mat{-5\\4\\-3}$
+			\item $\mat{1\\2\\3}$ and $\mat{-1\\-1\\2}$
+			\item $\mat{2\\4\\6}$ and $\mat{\frac{3}{2}\\3\\\frac{9}{2}}$
+			\item $\mat{0\\1\\0\\1}$ and $\mat{-1\\1\\3\\0}$
+		\end{enumerate}
+		\prob Determine which of the following vectors are orthogonal to eachother.
+		\begin{enumerate}
+			\item $\mat{0\\1}$ and $\mat{1\\0}$
+			\item $\mat{1\\2}$ and $\mat{1\\-2}$
+			\item $\mat{1\\3\\5}$ and $\mat{-3\\6\\-4}$
+			\item $\mat{1\\3\\5}$, $\mat{-3\\6\\-3}$, and $\mat{-13\\-4\\5}$
+			\item $\mat{1\\0\\0\\1}$ and $\mat{0\\1\\1\\\frac{1}{2}}$
+		\end{enumerate}
+
+		\prob Compute the distance between the following vectors
+		\begin{enumerate}
+			\item $\mat{-1\\1}$ and $\mat{-1\\-4}$
+			\item $\mat{2\\-6\\5}$ and $\mat{-4\\7\\-3}$
+			\item $\mat{1\\1\\1}$ and $\mat{-1\\-1\\-1}$
+			\item $\mat{6\\0\\2\\1}$ and $\mat{0\\1\\3\\-1}$
+			\item $\mat{0\\0\\0\\0\\0}$ and $\mat{1\\1\\1\\1\\1}$
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Which vector out of $\mat{1\\0}$, $\mat{0\\1}$, and $\mat{4\\1}$
+			has a direction closest to $\mat{3\\5}$?
+			\item Which vector out of $\mat{2\\3\\4}$, $\mat{1\\-1\\-1}$, and $\mat{-3\\0\\1}$
+			has a direction closest to $\mat{1\\0\\1}$?
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Find vector form and normal form of the plane $\mathcal P$ passing
+			through the points $A=(2,0,0)$, $B=(0,3,0)$ and $C=(0,0,-1)$.
+			\item Find vector form and normal form of the plane $\mathcal Q$ passing
+			through the points $D=(1,1,1)$, $E=(1,-2,1)$ and $F=(0,12,0)$.
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Let $\mathcal A\subseteq \R^3$ be the plane passing through $\mat{0\\1\\1}$
+			and with normal vector $\mat{-1\\-1\\-1}$. Write $\mathcal A$ in vector form.
+			\item Let $\mathcal B\subseteq \R^3$ be the plane passing through $\mat{1\\2\\3}$
+			and with normal vector $\mat{1\\-1\\0}$. Write $\mathcal B$ in vector form.
+		\end{enumerate}
+
+		\prob Find unit vectors in the same direction as the following vectors.
+		\begin{enumerate}
+			\item $\mat{90\\0}$
+			\item $\mat{16\\4}$
+			\item $\mat{11\\0\\13}$
+			\item $\mat{3\\5\\9}$
+			\item $\mat{0\\1\\0\\-1}$
+			\item $\mat{434\\1337\\1999\\4568}$
+		\end{enumerate}
+
```

Comment:
How about adding a conceptual problem where students derive the fact that the dot product is linear.